ice.c

```diff
@@ -4784,6 +4800,14 @@ void janus_ice_relay_rtp(janus_ice_handle *handle, janus_plugin_rtp *packet) {
 		extheader->length = 0;
 		/* Iterate on all extensions we need */
 		char *index = extensions + 4;
+		/* Check if we need to add the abs-send-time extension */
+		if(packet->video && handle->stream->abs_send_time_ext_id > 0) {
+			*index = (handle->stream->abs_send_time_ext_id << 4) + 2;
```

Comment:
+ 3 (24 bits), no?

Reply:
No, the length in the protocol field is always `actual data length -1` when you write the extension.

Reply:
In this case we're writing the first byte of the extension, so the ID of the extension and the length field:

> The 4-bit length is the number minus one of data bytes of this header extension element following the one-byte header.